sway-types/src/ident.rs

```diff
@@ -81,6 +83,18 @@ impl BaseIdent {
 /// often be different.
 pub type Ident = BaseIdent;
 
+impl Serialize for Ident {
+    // Serialize an `Ident` struct with two fields: `to_string` and `span`.
```

Comment:
Maybe these were intended to be doc comments? 
```suggestion
    /// Serialize an `Ident` struct with two fields: `to_string` and `span`.
```